third_party/txt/tests/paragraph_unittests.cc

```diff
@@ -3989,7 +3988,7 @@ TEST_F(ParagraphTest,
   txt::ParagraphBuilderTxt builder(paragraph_style, GetTestFontCollection());
 
   txt::TextStyle text_style;
-  text_style.font_families = std::vector<std::string>(1, "Roboto");
+  text_style.font_families = {"Roboto", "Noto Naskh Arabic"};
```

Comment:
Does this make the test able to run on macos now as well?  I'm not sure why it was turned off for mac but if it was relying on a fallback arabic font that could have been why it was turned off.

Reply:
IIRC many of these tests run only on Linux because the Skia font manager for Apple platforms uses Core Text APIs that may not yield the same text metrics as the libraries used on Linux.